package.json

```diff
@@ -20,13 +20,14 @@
   "bugs": {
     "url": "https://github.com/andela/ah-the-immortals-frontend/issues"
   },
-  "jest": {},
+  "jest": {
+    "setupTestFrameworkScriptFile": "./setupEnzyme.js"
```

Comment:
why do we need this when we do not have `setupEnzyme.js` file

Reply:
Hello @koechkevin ,
We don't need it. Thanks for catching that. I have updated it now.